vendor/kes-to-eso/migrate-post.sh

```diff
@@ -0,0 +1,11 @@
+#! /bin/bash -e
+KES_NAMESPACE="vault"
+
+mkdir -p eso_files
+
+# Generate ESO files and apply them
+./kes-to-eso generate -i kes_files -o eso_files -n $KES_NAMESPACE
```

Comment:
It seems that the binary needs to be compiled for different architecture. I put new version

Reply:
Hmmm. I downloaded amd64

Reply:
new version uploaded